.pre-commit-config.yaml

```diff
@@ -26,6 +26,12 @@ repos:
       - id: black
         language_version: python3
         args: ['--target-version', 'py37']
+  - repo: https://github.com/RobertCraigie/pyright-python
+    rev: v1.1.336
+    hooks:
+      - id: pyright
+        language_version: python3
+        args: ['--project', './pyrightconfig.json']
```

Comment:
TODO: delete line along with pyrightconfig.json, since args are not needed, as it reads the config in `pyproject.toml` instead